packages/remark-math/inline.js

```diff
@@ -2,72 +2,124 @@ function locator (value, fromIndex) {
   return value.indexOf('$', fromIndex)
```

Comment:
Not directly relevant to this PR, but I think it's worth noting that with the way this [locator][locator] thing works, if we wanted to do something like allow using `\[`/`\]` as delimiters as KaTeX does, this would prevent that from working. So we can't just pass in arbitrary left and right delimiters, they have to at least start with dollar signs.

[locator]: https://github.com/remarkjs/remark/blob/remark-parse%405.0.0/packages/remark-parse/readme.md#tokenizerlocatorvalue-fromindex